data/incipias/outfits.txt

```diff
@@ -0,0 +1,71 @@
+outfit "Plasma Discharger"
+	category "Guns"
+	mass 25
+	"outfit space" -20
```

Comment:
Is it deliberate that this doesn't use any weapon capacity?

Reply:
Ahem no